### PR TITLE
EDU-41 favicon

### PR DESCRIPTION
**Pull request by DylanMcKenzie**

### CODE REVIEW

**Review by dandabs — CHANGES_REQUESTED** [review]
prettify tsx doesn't seem to be working in the action

**Comment by JakeyGilly** [comment]
> 41

i dont think its working in anything because frontend doesnt look prettified

**Comment by dandabs** [comment]
> > prettify tsx doesn't seem to be working in the action
> 
> 
> 
> i dont think its working in anything because frontend doesnt look prettified

yeah? this ticket is meant to fix it - look at the code

**Comment by JakeyGilly** [comment]
> > > prettify tsx doesn't seem to be working in the action
> > 
> > 
> > i dont think its working in anything because frontend doesnt look prettified
> 
> yeah? this ticket is meant to fix it - look at the code

oh

File: packages/frontend/public/resources/images/favicon/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "MyWebSite",
```
Comment (dandabs) [inline packages/frontend/public/resources/images/favicon/site.webmanifest:2]: mywebsite?

**Review by dandabs — CHANGES_REQUESTED** [review]

**Comment by dandabs** [comment]
@DylanMcKenzie eta on this ticket?

**Pull request merged**